CHANGELOG.md

```diff
@@ -17,21 +17,25 @@ Here is a template for new release sections
 -
 ```
 
-# [Developer version] - 2019-01-13
+## [Developer version] - 2020-MM-DD
 
 ### Added
 - LICENSE.md with GPL v2.0 (#38, smartie2076)
 - folder "docs" and content to generate readthedocs (#39, smartie2076)
 - Started readthedocs homepage (not working): https://readthedocs.org/projects/mvs-eland/ (#39, smartie2076, #57, Bachibouzouk)
+- new feature to create the input json file from a collection of csv files (@Piranias)
+- new module added: A1_csv_to_json.py (@Piranias)
```

Comment:
```suggestion
- new module added: A1_csv_to_json.py, currently not called by main function mvs_eland_tool.py (@Piranias)
```

Reply:
I included into the main.py, see comment below